eng/tools/generator/typespec/tspconfig.go

```diff
@@ -117,6 +117,10 @@ func ParseTypeSpecConfig(tspconfigPath string) (*TypeSpecConfig, error) {
 	return &tspConfig, err
 }
 
+func (tsc *TypeSpecConfig) GetRelativePath() string {
+	return tsc.Options["@azure-tools/typespec-go"].(map[string]interface{})["service-dir"].(string) + "/" + tsc.Options["@azure-tools/typespec-go"].(map[string]interface{})["package-dir"].(string)
```

Comment:
the path could be multiple layer (>2) such as `azkeys`: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/security/keyvault/azkeys. could you ensure it could work?

Reply:
It works
![image](https://github.com/user-attachments/assets/843b24cc-9783-45fb-ab37-8072e3fb9ac1)
![image](https://github.com/user-attachments/assets/30379fa6-4482-4f0b-8100-d0ea4da473cd)
![image](https://github.com/user-attachments/assets/d4721f63-fea8-45bc-b2e7-2a6f186450c1)